1228-landing_page/landing_page_authenticated_user.feature

```diff
@@ -9,4 +9,13 @@ Feature: Authenticated User opens a Resource's Landing Page
 	Scenario: Authenticated User claims ownership of a Resource
 		When the User claims ownership of a Resource
 		Then a Ownership Request is created
-		And it contains a Message where the User presents their claim
\ No newline at end of file
+		And it contains a Message where the User presents their claim
+
+	Scenario: Privileged user sees Edit button for Resource
+		When the User is a validated contributor on the Resource
+		And the User has an active institusjon
+		Then the User has access to an Edit option
```

Comment:
Så alle bidragsytere skal kunne endre en registrering? både publisert og ikke-publisert, og for alle arbeidsflyter?